src/ftxui/screen/terminal.cpp

```diff
@@ -48,25 +48,38 @@ Dimensions& FallbackSize() {
   return g_fallback_size;
 }
 
-const char* Safe(const char* c) {
-  return (c != nullptr) ? c : "";
-}
-
 bool Contains(const std::string& s, const char* key) {
   return s.find(key) != std::string::npos;
 }
 
+// https://github.com/gabime/spdlog/blob/885b5473e291833b148eeac3b7ce227e582cd88b/include/spdlog/details/os-inl.h#L566
```

Comment:
I don't see any warning on my bots:
https://github.com/ArthurSonzogni/FTXUI/actions/runs/9695462958/job/26755276508

What configs should I add to reproduce?


Are you using the build config the FTXUI project define (e.g. are you using CMake at the end)

Reply:
Oh, I'm sorry. This is not msvc, but clang for windows.

```bash
cmake -Bbuild-clang -DCMAKE_CXX_COMPILER=clang++.exe -DCMAKE_C_COMPILER=clang.exe -GNinja
```

Reply:
@ArthurSonzogni see workflow run on my fork

[https://github.com/alexv-ds/FTXUI/actions/runs/10021288139/job/27699782082#step:6:72](https://github.com/alexv-ds/FTXUI/actions/runs/10021288139/job/27699782082#step:6:72)